services/slack_grid/precheck.go

```diff
@@ -0,0 +1,30 @@
+package slack_grid
+
+import (
+	"archive/zip"
+)
+
+func (t *BulkTransformer) GridPreCheck(zipReader *zip.Reader) bool {
+
+	requiredFiles := []string{
+		// "org_users.json",
```

Comment:
Does this still serve as purpose by being commented out?

Reply:
It doesn't, i'll remove it